docs/fastdds/ros2/ros2_configure.rst

```diff
@@ -129,8 +100,8 @@ using an XML file in combination with environment variable ``RMW_FASTRTPS_USE_QO
 Example
 -------
 
-The following example uses the ROS 2 talker/listener demo, configuring *Fast DDS* to publish synchronously,
-and to have a dynamically allocated publisher and subscriber histories.
+The following example uses the ROS 2 talker/listener demo, configuring *Fast DDS* to publish synchronously, and to have
+dynamically allocated publisher and subscriber histories.
```

Comment:
I'd complete the XML snippet so it's completely functional

Reply:
Any of the examples along the documentation has a whole functional XML. I am not sure what do you ask for here.

Reply:
I have added the `<profile>` tag. But along the documentation, most of the `profile` tags are commented in `XMLTester.xml`, so I see it as incongruity.